keadm/app/cmd/cloud/init.go

```diff
@@ -36,6 +36,16 @@ install and execute on the host.
 `
 	cloudInitExample = `
 kubeedge init
+
+- This command will download and install the default version of pre-requisites and KubeEdge
+
+kubeedge init --docker-version= --kubeedge-version=0.2.1 --kubernetes-version=1.14.1
```

Comment:
can we remove empty flag ? or add a value please ?
Applicable to similar places

Reply:
Done. Thanks